orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/GermlineFindingsChapter.java

```diff
@@ -107,6 +109,13 @@ private void addGermlineDeletions(@NotNull Document document) {
             String title = "Potentially pathogenic germline deletions (" + reportableGermlineGainsLosses.size() + ")";
             document.add(GainLossTable.build(title, contentWidth(), reportableGermlineGainsLosses, report.isofox(), reportResources));
```

Comment:
I know you haven't touched this code, but would look cleaner to rename variable to `reportableGermlineFullLosses` here?

Reply:
Done